### DISCUSSION TITLE
404 Error - PleaseHelp

### DISCUSSION THREAD

**Discussion by Ammarki** [community/community#83910] [New to GitHub]
Hello everyone,

I am starting to develop my own website, and just tested it out. 404 error is what I get.

I have been trying different ways to get to the bottom of the issue, with no luck. I started facing the issue when I switched my repository from public to private, and back to public again. When I did, I had to copy the whole code into the README file. 
Here is a link to my repository: https://github.com/Ammarki/ammarKi.github.io.git

I would be so grateful if you could guide me to where the issue might be.

Cheers,,

Ammar

**Comment by shivakr07** [comment]
solution1 : you can push the code again
solution2 : reset it to the default
